common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -697,6 +697,24 @@
   public static final String SUNBIRD_USER_PROFILE_FIELD_DEFAULT_VISIBILITY =
       "sunbird_user_profile_field_default_visibility";
   public static final String DEFAULT_PROFILE_FIELD_VISIBILITY = "defaultProfileFieldVisibility";
+  public static final String COURSE_BATCH_NOTIFICATIONS_ACTIVE = "course_batch_notification_active";
```

Comment:
will "course_batch_notification_active" come from env , if yes then update to "sunbird_course_batch_notification_active"